test/test_autograd.py

```diff
@@ -5624,18 +5624,18 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
 
         def context_fn():
             return verbose_mode, contextlib.nullcontext()
-        out = checkpoint(lambda x: x.sin(), x, use_reentrant=False, context_fn=context_fn)
-        self.assertEqual(verbose_mode.operators, ['sin.default'])
+        out = checkpoint(lambda x: x.exp(), x, use_reentrant=False, context_fn=context_fn)
```

Comment:
Since this test is supposed to test usage of the `VerboseMode`, making it `exp` makes the recorded aten ops for the forward and recomputation look more similar. If it were still `sin`, since sin saves the input, recomputation is not needed at all, so nothing will show up in the trace.

(I suppose I could've disabled early stop as well.)